frontend/src/metabase/home/components/CustomHomePageModal/CustomHomePageModal.tsx

```diff
@@ -107,8 +109,13 @@ export const CustomHomePageModal = ({
         <DashboardSelector
           value={dashboardId}
           onChange={handleChange}
-          collectionFilter={(collection: Collection) =>
-            collection.personal_owner_id === null || collection.id === "root"
+          collectionFilter={(
+            collection: Collection,
+            _index: number,
+            allCollections: Collection[],
+          ) =>
+            !isPersonalCollectionOrChild(collection, allCollections) ||
+            collection.id === "root"
```

Comment:
Similar to the other comments - does this simplification make sense?

```suggestion
            !isPersonalCollectionOrChild(collection, allCollections)
```